### PR TITLE
make mmr-for-profile component

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: app/components/mmr-for-profile.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
```
Comment (NullVoxPopuli) [inline app/components/mmr-for-profile.js]: This is the actual change here.

Comment (NullVoxPopuli) [reply app/components/mmr-for-profile.js]: (the file)

File: app/models/profile.js [hunk 2]
```diff
@@ -15,26 +15,6 @@ export default Model.extend(LoadableModel, {
 
   knownLadders: DS.attr('number'),
 
-  soloMMR: computed('snapshot', 'ladderSummary.isFulfilled', 'ladderSummary.ladders.isFulfilled', function() {
```
Comment (NullVoxPopuli) [inline app/models/profile.js]: this isn't needed because it's handeled by the new component

**Comment by NullVoxPopuli** [comment]
Some notes:
 - models should be concerned with mapping api data only.
   - any computations on models, unless they are directly tied to the api data, should probably be in helper functions (app/utils), or computed properties where needed.
      - the mmr thing is really the only computed property that violated this.
   - try to keep models more focused on representation of data, rather than how the data will be displayed on the UI (cause that can change, esp if you want to add i18n)
 
- most of this PR is octane upgrades
- think of components like functions --- keep them small, use them liberally -- "do one thing, do it well"

File: app/components/mmr-for-profile.js (hunk 1 above)
Comment (AlbinoGeek) [reply app/components/mmr-for-profile.js]: woah, that's much cleaner

**Pull request merged**